internal/v1/api.yaml

```diff
@@ -483,6 +483,14 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/HTTPErrorList'
+    delete:
+      summary: delete a blueprint
+      description: |
+        Deletes all versions of Blueprint, the compose will still count towards quota.
+      operationId: deleteBlueprint
+      responses:
+        204:
+          description: OK
```

Comment:
What about 404 here? :thinking: Which leads me to the fact that we do not really test 404 in handlers, right?

Reply:
Also, can you possibly add the following?

```
      tags:
        - blueprint
```

Reply:
that's true, good catch, let me add 404 and test for it here :+1:
We can revisit other handlers in follow-ups :)